src/test/java/org/drugis/trialverse/dataset/repository/DatasetReadRepositoryTest.java

```diff
@@ -280,7 +278,7 @@ public void testExecuteVersionedQuery() throws URISyntaxException, IOException {
     String versionUuid = "http://myVersion.ninja";
     String acceptHeader = "confirm/deny";
     URI datasetUrl = new URI(Namespaces.DATASET_NAMESPACE + datasetUuid);
-    VersionMapping versionMapping = new VersionMapping(1, "http://whatever", "pietje@precies.gov", datasetUrl.toString());
+    VersionMapping versionMapping = new VersionMapping(1, "http://whatever", "pietje@precies.gov", datasetUrl.toString(), false, null);
```

Comment:
Fix this with a constructor that doesn't need archived status imo

Reply:
There is, for cases without ID. This fringe case only happens in very few places.